conan/cli/formatters/audit/vulnerabilities.py

```diff
@@ -38,15 +37,15 @@ def wrap_and_indent(txt, limit=80, indent=2):
         lines.append(" " * indent + txt)
         return "\n".join(lines)
 
-    if not response or "data" not in response or not response["data"]:
+    if not result or "data" not in result or not result["data"]:
         if not errors_in_response:
             cli_out_write("No vulnerabilities found.\n", fg=Color.BRIGHT_GREEN)
```

Comment:
This is different, this shouldn't be using the ``conan_error`` data, that is for raising an exception. There are vulnerabilities below the threshold, but ``conan_error`` will not be defined.

Reply:
Fixed by #18171